### DISCUSSION TITLE
How to reference Width of a button from another button (XAML window)

### DISCUSSION THREAD

**Discussion by washburnit** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2706] [Q&A] [answered]
Hi,

first I'm new with WPF and MaterialDesign XAML so I apologize if my question seems silly.
I'm developing an app  where I have several buttons inside a "DrawerHost".
I would like to know, if possibile, how to reference, for example, the Width of a button from another button: let's say I have Button1 and Button2.
Button1 Width is "50", how can I reference the same Width for Button2?
Something like this:

```
<Button x:Name="Button1" Style="{StaticResource MaterialDesignFlatLightBgButton}"
                            Margin="0,20"  Width="200" Height="45" Cursor="Hand">
</Button>

<Button x:Name="Button2" Style="{StaticResource MaterialDesignFlatLightBgButton}"
                            Margin="0,5"  Width="Button1.Width" Height="45" Cursor="Hand">
</Button>
```

How can I do it?
Of course I could do it at runtime into the code but I wouldn't have a "visual" feedback on the designer.
Thank you, regards.

Roberto

**Comment by ElieTaillard** [comment, marked as answer]
```
<Button x:Name="Button1" Style="{StaticResource MaterialDesignFlatLightBgButton}"
                            Margin="0,20"  Width="200" Height="45" Cursor="Hand">
</Button>

<Button x:Name="Button2" Style="{StaticResource MaterialDesignFlatLightBgButton}"
                            Margin="0,5"  Width="{Binding Path=ActualWidth, ElementName=Button1}" Height="45" Cursor="Hand">
</Button>
```

**Comment by washburnit** [comment]
Great, thank you.
Regards,

Roberto
